apps/src/templates/rubrics/RunAIAssessmentButton.jsx

```diff
@@ -169,7 +171,7 @@ export default function RunAIAssessmentButton({
             color={Button.ButtonColor.neutralDark}
             onClick={handleRunAiAssessment}
             style={{margin: 0}}
-            disabled={status !== STATUS.READY}
+            disabled={status !== STATUS.READY && status !== STATUS.ERROR}
```

Comment:
I couldn't see in the video, but according to this it's still disabled after a successful run, right?

Reply:
correct. that is the `SUCCESS` state

![Screenshot 2024-04-05 at 1 35 54 PM](https://github.com/code-dot-org/code-dot-org/assets/8001765/a838b972-b363-4427-bc27-da15df0941c4)

Reply:
sorry, the screenshot I showed is for the `ALREADY_EVALUATED` state, but I did previously confirm this is also disabled for the `SUCCESS` state which we show after evaluation succeeds.